app/src/main/java/com/color_wave/ColorWaveRestClient.java

```diff
@@ -3,7 +3,7 @@
 import com.loopj.android.http.*;
 
 public class ColorWaveRestClient {
-    private static final String BASE_URL = "http://10.0.2.2:8000/";
```

Comment:
This endpoint is the production endpoint and you're changing it back to a non-existing one